src/ngx_http_lua_kong_log_handler.c

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019-2023 Kong Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *    http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ngx_http_lua_kong_common.h"
+
+
+void
+ngx_http_lua_kong_error_handler(ngx_http_request_t *r, u_char **buf, size_t *len)
+{
+    u_char                       *p;
+    ngx_http_lua_kong_loc_conf_t *lcf;
+    ngx_http_complex_value_t     *cpx_ela;
```

Comment:
`cpx_ela` is hard to understand, could we use meaningful name instead of acronym？